csv_detective/detection.py

```diff
@@ -2,9 +2,14 @@
 import numpy as np
 from cchardet import UniversalDetector
 from ast import literal_eval
+import logging
+from time import time
+from csv_detective.utils import print_colored_logs
 
 
-def detect_continuous_variable(table, continuous_th=0.9):
+logging.basicConfig(level=logging.INFO)
```

Comment:
can you variabilize this. With a `LOG_LEVEL` variable for instance. With a default to `logging.INFO`if not set.

Reply:
This is set in three different files, I can set it in one file only and import it in the others. Is this what you mean?